blocked-edges/4.11.36-PatchesOlderRelease.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.11.36
+from: 4[.]11[.]35
+url: https://access.redhat.com/solutions/7007136
+name: PatchesOlderRelease
+message: |-
+  4.11.36 patches 4.11.34 to fix https://issues.redhat.com/browse/OCPBUGS-11663 , which is an installer issue, and not and issue with running clusters.  A number of other changes had landed since 4.11.34 and were shipped in 4.11.35.  We expect those fixes to be useful, but they might also introduce regressions, and because they have not soaked for as long, we did not include them in 4.11.36.  But that means we know 4.11.36 has regressed on all of those bugs compared to 4.11.35, so we do not recommend updating 4.11.35 clusters to 4.11.36.  Later 4.11.z will include both 4.11.36's installer fix and 4.11.35's fixes, and be recommended update targets for 4.11.35.
```

Comment:
No worries from me if folks would rather explain the context in the KCS, or a new KCS, or a new Jira ticket, or whatever, and say something shorter like:

> We do not recommend updating 4.11.35 clusters to 4.11.36 to avoid multiple, intentional regressions.  4.11.34 and earlier are not exposed to these regressions.  Later 4.11.z will be recommended update targets for 4.11.35.